server/fleet/policies.go

```diff
@@ -209,3 +209,25 @@ func (p PolicySpec) Verify() error {
 	}
 	return nil
 }
+
+// FailingPolicySet holds sets of hosts that failed policy executions.
+type FailingPolicySet interface {
```

Comment:
Nit: This doesn't have to be called `FailingPolicySet`, it could be called `WebhookTargetSet` or something more generic?